xarray/core/common.py

```diff
@@ -346,6 +346,28 @@ def assign_coords(self, **kwargs):
         data.coords.update(results)
         return data
 
+    def assign_attrs(self, *args, **kwargs):
+        """Assign new attrs to this object, returning a new object equivalent
```

Comment:
Can you move the second half of this to a new line? We try to keep the title to one line.